app/src/main/res/values/strings.xml

```diff
@@ -487,9 +487,9 @@
     <string name="unable_to_retrieve">Unable to retrieve</string>
     <string name="blocks">Blocks</string>
     <string name="unblock">Unblock</string>
-    <string name="blocked_users">Blocked users</string>
-    <string name="blocked_communities">Blocked communities</string>
-    <string name="blocked_instances">Blocked instances</string>
+    <string name="blocked_users">Users</string>
```

Comment:
You changed english translation but now it no longer correct. Name doesn't correspond and existing translations either.

Put this back, if not used remove including translations. Check if existing entries exist. If not create ones.

Reply:
What do you mean by "name doesn't correspond"? Should I change the string res name from "blocked_users" to "users"? These resources are only used in the blocks screen.

The problem here is that the translations are now incorrect when compared to the english text?

Reply:
Yes, you only changed the English translation. I don't expect you fix the other languages, so instead create new entry just for Users, Communities, Instances. The blocked translations wont have any purpose anymore so you can then remove them including their translations

Reply:
Done. I removed the instances_xxx string resources and created new resources for this screen